packages/plugin-calendar/test/integration/spec/calendar.js

```diff
@@ -0,0 +1,143 @@
+
+    const startInterval = moment.utc().add(2, `hours`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`);
+    const iCalUidUUID = uuid.v4();
+    const locusId = uuid.v4();
+
+    before(`creates a meeting`, () => {
+      const meetingParams = {
+        changeType: `CREATE`,
+        userId: organizer.emailAddress,
+        userUUID: organizer.id,
+        emailAddress: organizer.emailAddress,
+        itemId: (`Item_ID-`).concat(iCalUidUUID),
+        meeting: {
+          attachments: [],
+          body: `Test Agenda`,
+          callUri: (`https://locus-example.com/locus/api/v1/loci/`).concat(locusId),
+          encryptionV2: true,
+          endDate: moment.utc(startInterval).add(1, `hours`).format(`YYYY-MM-DD[T]HH:mm:ss.SSS[Z]`),
+          featureTogglesMap: [
+            {
+              calsvc_calendar_view: true
+            }
+          ],
+          hasAttachments: false,
+          iCalUid: iCalUidUUID,
+          isRecurring: false,
+          isReminderSet: true,
+          itemId: (`Item_ID-`).concat(iCalUidUUID),
+          location: `@webex`,
+          mailboxAttendeesInMeeting: [
+            mccoy.emailAddress,
+            spock.emailAddress
+          ],
+          mailboxInviteesInMeeting: [
+            {
+              id: mccoy.id,
+              invitee: mccoy.emailAddress,
+              type: `PERSON`
+            },
+            {
+              id: spock.id,
+              invitee: spock.emailAddress,
+              type: `PERSON`
+            }
+          ],
+          meetingIdentifier: {
+            id: (`Meeting_ID-`).concat(iCalUidUUID)
+          },
+          meetingTime: startInterval,
+          meetingTimeZone: `(UTC-08:00) Pacific Time (US & Canada)`,
+          organizer: organizer.emailAddress,
+          startDate: startInterval,
+          subject: (`test-plugin-calendar_#meetings_`).concat(iCalUidUUID)
+        },
+        meetingIdentifier: {
+          id: (`Meeting_ID-`).concat(iCalUidUUID)
+        }
+      };
+
+      return spark.request({
+        method: `POST`,
+        service: `whistler`,
+        resource: `calendarNotification`,
+        body: meetingParams,
+        qs: {
+          isinteg: false
+        }
+      })
+        .then((res) => {
+          createdMeeting = res.body;
+          console.log(`Created Meetings:\n `, JSON.stringify(createdMeeting, null, 4), `\n`);
+          assert.isDefined(createdMeeting);
+          assert.equal(createdMeeting.meeting.meetingIdentifier.id, meetingParams.meetingIdentifier.id);
+          assert.equal(createdMeeting.userUUID, meetingParams.userUUID);
+          assert.equal(createdMeeting.meeting.startDate, meetingParams.meeting.startDate);
+          assert.equal(createdMeeting.meeting.endDate, meetingParams.meeting.endDate);
+        })
+        .catch((err) => {
+          this.logger.error(`test-creates a meeting: error occurred in creating meeting`, err);
```

Comment:
You're suppressing the error here, which means the it() statements will run and fail strangely. Don't catch this error, let mocha log it when it happens